### DISCUSSION TITLE
Create 404.html error file with Jekyll

### DISCUSSION THREAD

**Discussion by salimbenfarhat** [community/community#45586] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello, in Jekyll and with Github Pages, how it is possible to create custom 404 error file called 404.html in order to add HTML code instead of Markdown.

**Comment by jge162** [comment, marked as answer]
Hello @salimbenfarhat Here are some steps you can take to create the 404 error file. 

a. create a new file in your Jekyll project called 404.html. This file should be located in the root directory of your project.

b. in this file, you can add your custom HTML code. You can also include Jekyll liquid tags to include dynamic content.

c. in your _config.yml file, add the following line: permalink: /404.html

d. this will ensure that Jekyll generates a 404.html file in your site's root directory.

e. commit the changes and push the code to your GitHub repository.

f. go to your GitHub Pages repository settings, and set the custom 404 page to your 404.html file.

then go ahead and load website to see.

**Reply by salimbenfarhat** [reply]
Thank you @jge162
